src/modules/item/Item.tsx

```diff
@@ -503,12 +502,28 @@ const Item = ({
 }: Props): JSX.Element | false => {
   const { t: translateMessage } = useMessagesTranslation();
   const { data: item, isInitialLoading: isLoadingItem, isError } = useItem(id);
+  const { rootId } = useParams();
+  const { data: root } = useItem(rootId);
 
   if (item && item.type === ItemType.FOLDER) {
     if (isChildren) {
-      return <ItemContentWrapper item={item} />;
+      return (
+        <>
+          <ItemContentWrapper item={item} />;
+        </>
+      );
     }
-    return <FolderContent item={item} showPinnedOnly={showPinnedOnly} />;
+
+    return (
+      <>
+        <Helmet>
+          <title>
+            {rootId !== id ? `${root?.name} | ` : ''} {item.name}
```

Comment:
This is not very clear, make it more explicit with an `if-else` condition maybe inside a new component. 

Also I think we want to have the item name first, as this is the most important information. 

Did you look at how other sites name their tabs ?